meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,297 @@
+
+Given the following dataset:
+```json
+{
+  "id": 0,
+  "name": "Our Wives Under the Sea",
+  "genre": ["horror", "scifi"],
+  "synopsis": "A woman returns to her wife transformed after a deep-sea adventure."
+},
+{
+  "id": 1,
+  "name": "A Strange and Stubborn Endurance",
+  "genre": ["adventure"],
+  "synopsis": "A man must overcome trauma and fight off a mysterious assassin."
+}
+```
+And the following query:
+```json
+{
+  "q": "adventure",
+  "attributesToSearchOn": ["genre"]
+}
+```
+Meilisearch will only return document 1.
+
+Both documents contain the word adventure, but `"attributesToSearchOn": ["genre"]` instructs Meilisearch to only consider results found on the genre field.
+
+
+Done by @ManyTheFish in (#3834)
+
+
+## Search for facet values
+Meilisearch v1.3.0 introduces a new endpoint  `POST /indexes/{index}/facet-search` that allows you to search for facet values. Only fields defined as `filterableAttributes` will be facet-searchable. Facet search supports [prefix search](https://www.meilisearch.com/docs/learn/advanced/prefix) and [typo tolerance](https://www.meilisearch.com/docs/learn/configuration/typo_tolerance).
+
+```bash
+ curl \
+  -X POST 'http://localhost:7700/indexes/movies/facet-search' \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "facetName": "genres",
+    "facetQuery": "a"
+  }'
+
+```
+
+Done by @Kerollmops in (#3699)
+
+## Sort facet values by count
+By default, all facet values are sorted in ascending alphanumeric order. 
+```json
+"sortFacetValuesBy": {"*": "alpha"}
+```
+
+With the release of v1.3.0 you can now order by `count` using the `sortFacetValuesBy` property of the [`faceting` index settings](https://www.meilisearch.com/docs/reference/api/settings#faceting). This allows you to sort facet values in descending order by the number of documents containing that facet value.
+
+It is possible to change this ordering for all facets using `*`:
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings/faceting \
+  -H 'Content-Type: application/json'  \
+  --data-binary '{
+    "sortFacetValuesBy": {"*": "count"}
+  }'
+
+```
+
+Alternatively, it is possible to specify ordering per facet i.e. order the genre facet by count, for everything else apply alphanumeric ordering.
+
+```bash
+ curl \
+  -X PATCH 'http://localhost:7700/indexes/movies/settings/faceting \
+  -H 'Content-Type: application/json'  \
+--data-binary '{
+    "sortFacetValuesBy": {"*": "alpha", "genre": 'count'}
+  }'
+
+```
+
+Done by @Kerollmops in (#3612)
+
+### Language support improvements
+
+* Enhance Japanese words segmentation (#218) @mosuka
+* Enhance Separator based segmentation (#215) @ManyTheFish 
+
+Done by @ManyTheFish, @mosuka in [#] and in [Charabia v0.7.3](https://github.com/meilisearch/charabia/releases/tag/v0.7.3)
```

Comment:
@ManyTheFish is this going to be version v0.8.0 rather than 0.7.3? 
I just saw: https://github.com/meilisearch/charabia/pull/221 but there also seems to be a 0.7.3 one so I just want to be sure

Reply:
Yes there are some breaking changes in the Charabia API :)

Reply:
thanks! should we mention details about that in the changelogs here?

Reply:
Nope, the changes in the Charabia API does not impact the meilisearch users. The Charabia library is breaking, but not Meilisearch
Let me know if I'm wrong @ManyTheFish